src/net.cpp

```diff
@@ -575,6 +578,9 @@ CNode* CConnman::ConnectNode(CAddress addrConnect, const char *pszDest, bool fCo
                              });
     pnode->AddRef();
 
+    // update connection count by network
+    if (pnode->IsManualOrFullOutboundConn()) { m_network_conn_counts[addrConnect.GetNetwork()] += 1; }
```

Comment:
style: remove the braces or put the statement on a new line. From `doc/developer-notes.md`:

>  - If an `if` only has a single-statement `then`-clause, it can appear
    on the same line as the `if`, without braces.

(same when decrementing)